### ISSUE TITLE
Docker build broken

### ISSUE THREAD

**Issue by odin568** [darkphnx/ical-filter-proxy#23]
Hi!
Just saw you are activly developing on this component and updated dependencies. Also got the point about Python 3.
Just wanted to play around with the regex changes (#19 ) and tried to use the docker build for it as I did yesterday before your changes.
Docker build says 
`#8 0.367 You must use Bundler 2 or greater with this lockfile.`
When I update to 2.7 ruby container (what might make sense in general?) I can build but then get when starting the container:
```
bundler: command not found: rackup
Install missing gem executables with `bundle install`
```
Can you have a look into this? 

In general, would it make sense to add a docker build into your project and push the container to dockerhub? This would be finally a requirement for me, would need to fork your project for it for that purpose only...

Anyhow already thanks for your work - exactly what I need!

**Comment by darkphnx** [comment]
Will look into this today.

**Comment by darkphnx** [comment]
Fixed in #25

**Comment by darkphnx** [comment]
Can be found on dockerhub at https://hub.docker.com/r/darkphnx/ical-filter-proxy
